styles.css

```diff
@@ -341,3 +381,113 @@ textarea:focus {
 #submit-input {
   margin: 0.3rem 0 0;
 }
+
+@media screen and (min-width: 48em) {
```

Comment:
- [ ] Your project is awesome 🥇. But let's fix a little issue 😉. Your media query for the Desktop breakpoint is too small. The value for `min-width` property should be in range of `992px - 1100px`. However, yours is `48em` which is equivalent to `768px`. Kindly increase your min-width to be in the range of 992px - 1100px. Hint; 1em = 16px.